File: java_work/ex0708/src/ex0708/Accumulator.java
package ex0708;

public class Accumulator {

	class Add {
		int i = 0;
		int sum = 0;
		public Add(int i, int sum) {
			sum += i;
		}
		public showResult() {
			System.out.println();
		}
	}
	
	public static void main(String[] args) {
		Add add = new Add();
		for(int i=0;i<10;i++) {
			Accumulator.add(i);
			
		Accumulator.showResult();
		}
	}
}

File: python_work/220711/accumulator.py
# sum = 0

class Accumulator:
    def __init__(self):
        self.sum = 0
    # @staticmethod
    def add(self,i):
        # global sum #global: 밖에 선언된 전역변수를 사용하겠다
        self.sum += i
    # @staticmethod
    def showResult(self):
        print(f"sum = {self.sum}") #python 3.ver에서만 사용가능